packages/lesswrong/server/tagging/updateDenormalizedHtmlAttributions.ts

```diff
@@ -1,9 +1,14 @@
-import { Tags } from '../../lib/collections/tags/collection';
 import { annotateAuthors } from '../attributeEdits';
+import { getCollection } from '../vulcan-lib';
 
-export async function updateDenormalizedHtmlAttributions(tag: DbTag) {
-  const html = await annotateAuthors(tag._id, "Tags", "description");
-  await Tags.rawUpdateOne({_id: tag._id}, {$set: {
+export async function updateDenormalizedHtmlAttributions(
+  document: any,
+  collectionName: CollectionNameString,
+  fieldName = 'description'
```

Comment:
A bit scary to have a default value here, since it's only legal for one of the collections